int16 setpoint_left
int16 setpoint_right
int16 ff_left
int16 ff_right
